Backend/routes/routes.py

```diff
@@ -1,8 +1,9 @@
 from argparse import OPTIONAL
-from fastapi import APIRouter
-from Backend.model.model import Ngo, User
-from Backend.config.database import NgoCollection, UserCollection
-from Backend.schemas.schema import ngo_list_serializer, user_list_serializer
+from fastapi import APIRouter,Body
+from Backend.model.model import Ngo, User, Donor
+#from Backend.config.database import NgoCollection, UserCollection
+from Backend.config.database import donors
```

Comment:
we can rename this donors to DonorsCollection instead of donors alone for consistency for now